apps/docs/src/.vitepress/components/StorybookIFrame.vue

```diff
@@ -12,12 +12,17 @@ const props = defineProps<{
    * @example MyInput
    */
   component: string;
+  /**
+   * Storybook folder name (lowercase) as defined in the meta.title property in the *.stories.ts file
+   * @example "components" for meta title "components/MyInput"
+   */
+  folderName: string;
 }>();
 
 const { isDark } = useData();
 
 const iframeSrc = computed(() => {
-  const id = `components-${props.component.toLowerCase()}--docs`;
+  const id = `${props.folderName}-${props.component.toLowerCase()}--docs`;
```

Comment:
We only should show the public components in vitepress, for everything else (support/internal) components we have the storybook

Reply:
But the Storybook is not linked anywhere yet. Should I add a link to the header inside the "Resources" dropdown instead?

Reply:
Sounds good :)

Reply:
Done